### PR TITLE
refactor(editor): RunData components to composition API (no-changelog)

### PR DESCRIPTION
**Pull request by elsmr**
## Summary

refactor RunData components to composition API

## Review / Merge checklist

- [ ] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/editor-ui/src/components/InputPanel.vue [hunk 1]
```diff
@@ -358,18 +335,19 @@ export default defineComponent({
...
 		@activate-pane="activatePane"
-		@item-hover="$emit('itemHover', $event)"
+		@item-hover="emit('itemHover', $event)"
 		@link-run="onLinkRun"
 		@unlink-run="onUnlinkRun"
 		@run-change="onRunIndexChange"
-		@table-mounted="$emit('tableMounted', $event)"
-		@search="$emit('search', $event)"
+		@table-mounted="emit('tableMounted', $event)"
+		@search="emit('search', $event)"
```
Comment (alexgrozav) [inline packages/editor-ui/src/components/InputPanel.vue:343]: nit: I'd extract these 3 into separate methods as well to align with `onLinkRun`, `onUnlinkRun`, etc.

File: packages/editor-ui/src/components/RunData.vue [hunk 2]
```diff
@@ -85,1135 +93,1091 @@ export type EnterEditModeArgs = {
...
+	() =>
+		workflowExecution.value?.status === 'waiting' &&
+		workflowExecution.value.data?.waitTill &&
+		workflowExecution.value?.data?.resultData?.lastNodeExecuted === node.value?.name,
+);
 
-			if (this.binaryData?.length) {
-				return this.isPaneTypeOutput;
-			}
+const activeNode = computed(() => ndvStore.activeNode);
```
Comment (alexgrozav) [inline packages/editor-ui/src/components/RunData.vue:197]:
```suggestion
const { activeNode } =  storeToRefs(ndvStore);
```

File: packages/editor-ui/src/components/RunDataTable.vue [hunk 3]
```diff
@@ -1,389 +1,380 @@
...
+const columnLimitExceeded = ref(false);
+const draggableRef = ref<DraggableRef>();
+
+const ndvStore = useNDVStore();
+const workflowsStore = useWorkflowsStore();
+
+const i18n = useI18n();
+const telemetry = useTelemetry();
+
+const hoveringItem = computed(() => ndvStore.hoveringItem);
```
Comment (alexgrozav) [inline packages/editor-ui/src/components/RunDataTable.vue:66]:
```suggestion
const { hoveringItem, focusedMappableInput, highlightDraggables: highlight } = storeToRefs(ndvStore);
```

**Review by alexgrozav — DISMISSED** [review]
Probably some of the hardest components to rewrite to composition API. Amazing work!

File: packages/editor-ui/src/components/InputPanel.vue [hunk 4]
```diff
@@ -10,331 +10,308 @@ import { useNDVStore } from '@/stores/ndv.store';
...
+	{ value: 'mapping', label: i18n.baseText('ndv.input.mapping') },
+	{ value: 'debugging', label: i18n.baseText('ndv.input.debugging') },
+];
+
+const nodeTypesStore = useNodeTypesStore();
+const ndvStore = useNDVStore();
+const workflowsStore = useWorkflowsStore();
+const uiStore = useUIStore();
+const focusedMappableInput = computed(() => ndvStore.focusedMappableInput);
+const isUserOnboarded = computed(() => ndvStore.isMappingOnboarded);
```
[the remark this answers is not in the document]
Comment (elsmr) [reply packages/editor-ui/src/components/InputPanel.vue]: Didn't know about this API, cool!

**Review by RicardoE105 — DISMISSED** [review]

**Review by CharlieKolb — APPROVED** [review]

**Review by alexgrozav — APPROVED** [review]

**Review by RicardoE105 — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.68.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.68.0)

**Pull request merged**